src/you_get/extractors/youku.py

```diff
@@ -254,6 +278,7 @@ def extract(self, **kwargs):
                     streamfileid = piece['fileid']
                     for no in range(0, len(segs)):
                         k = segs[no]['key']
+                        assert k != -1
```

Comment:
https://github.com/linhua55/you-get/blob/3be2f06877308a6ffb8d2592b6b4fa0881ea4f71/src/you_get/extractors/youku.py#L131-L142

Maybe related to this snippet.
We only deal with the `outside mainland china` error.There are some errors we don't handle.
And There may be some errors  related to `k = -1`.